lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -12,6 +12,7 @@
             <input matInput
                    *ngIf="!property.multiline"
                    class="adf-property-value"
+                   title={{property.displayValue}}
```

Comment:
it should be the title of the property, not the value
```
title={{property.label}}
```

and for other inputs as well

Reply:
title="{{property.label | translate }}" has been added for all input fields.

Reply:
Resolved some accessibility issues present in CSV file and working on the remaining one's

Reply:
Resolved some accessibility issues present in CSV file and working on the remaining one's